lib/pure/typetraits.nim

```diff
@@ -100,6 +100,16 @@ proc isNamedTuple*(T: typedesc): bool {.magic: "TypeTrait".} =
     doAssert not isNamedTuple((string, int))
     doAssert isNamedTuple(tuple[name: string, age: int])
 
+template pointerBase*[T](_: typedesc[ptr T | ref T]): typedesc =
+  ## Returns `T` for `ref T | ptr T`
```

Comment:
```suggestion
  ## Returns `T` for `ref T` or `ptr T`.
```

Reply:
i added the `.` but kept the `|` which is shorter and idiomatic